portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+/** A comment on a the history list. */
+public final class Comment {
+
+  private final long id;
+  private final String text;
+  private final long timestamp;
+
+  public Comment(long id, String text, long timestamp) {
```

Comment:
I know that the Task example use the variable name timestamp, but that it doesn't have a object meaning, so I would suggest update it to something as 'creation_time' or a similar one that you would prefer.

Reply:
![image](https://user-images.githubusercontent.com/44590322/85966262-cf1fa280-b984-11ea-9dfb-e59828fe3488.png)
Changed `timestamp` variable to `creationTime` for code clarification.